maths/largest_of_very_large_numbers.py

```diff
@@ -4,6 +4,23 @@
 
 
 def res(x, y):
+    """
+    Reduces large number to a more manageable number
+    >>> res(5, 7)
+    4.892790030352132
+
+    >>> res(0, 5)
+    0
+
+    >>> res(3, 0)
+    1
+
+    >>> res(-1, 5)
+    Traceback (most recent call last):
+    ...
+    ValueError: math domain error
+
```

Comment:
It is better if the tests do not require extra scrolling when someone is reading the code.  If they want to read all the tests, they can take the time to do so but often readers will want to look at the function signature and then jump straight to the code.  Keeping the tests in tight block will allow them them to study the function signature and the code without too much visual distraction.
```suggestion
    >>> res(0, 5)
    0
    >>> res(3, 0)
    1
    >>> res(-1, 5)
    Traceback (most recent call last):
    ...
    ValueError: math domain error
```